tools/nl/detection_evals/golden_curation/README.md

```diff
@@ -0,0 +1,30 @@
+# Golden Curation
+
+This folder contains resources and scripts for curating the goldens for nl
+detection evals. These are for auto-curating the goldens that then need to go
+through manual checks.
+
+## Generate Vars
+
+This auto-curates a list of variable dcids (including all the variables in the
+Data Commons knowledge graph) that match a given query.
+
+### Prerequisites
+
+1. In your input folder, you need to have your CSV files with the following columns (there can be multiple rows for the same query if there is disagreement on what the variable string for the query is)
+    - query_num: This is a unique number for each query
+    - query: This is the full query string
+    - var_string: This is the substring of the query that should be used to
+    detect the variable in the query
+    - groups: These are the groups that the variable in the query could belong to. The variable could belong to multiple groups and these groups should be comma separated.
+2. Start a local server with additional embedding indices enabled for all the variable groups
```

Comment:
is this a `website` server w/ llm_mode enabled? Can you add some more info about getting it started?
Is it simply adding the catalog + env configs you've linked and then running `./run_server -m` like normal?